app/src/docs/content/streams/usingStreamsInCore.mdx

```diff
@@ -32,10 +36,32 @@ Head over to the <Link to={routes.streams.index()}>Streams page</Link> to see an
   alt="Stream Listing"
 />
 
-This stream listing page gives an indication for when data was last pushed to each stream and uses traffic light indicators show the health of each stream, at a glance. Green shows a stream receiving data, while red indicates no data flowing. Clicking on a stream will open the stream editor. You may only edit streams that you have created or have explicit permission to edit.
+This stream listing page displays all streams the current user has a right to publish or subscribe to. Clicking on a stream will open the stream editor. You may only edit streams that you have created or have explicit permission to edit.
 
 By clicking on the more options icon you may access advanced features for each stream, such as stream sharing and code snippets. A new stream can be created by the clicking the Create Stream button.
 
+### Creating streams
+Streams are identified by a stream ID, which is an Ethereum address plus a user-given path, just like an Internet URL. The Ethereum address is referred to as the domain and is either the Ethereum address of the logged in user, or it can also be the ENS (Ethereum Name Service) name owned by the Ethereum address of the logged in user. All streams require a unique path in the format of domain/pathname and can contain single or multi-level paths.
+
+ENS names allow for completely human readable names, for example `mydomain.eth/traffic/helsinki`. If you haven’t used an ENS name before, you can easily create one with a small amount of ETH on <Link to={routes.misc.ensDomains()}>ENS Domains</Link>. Your purchased addresses will automatically appear in the stream editor for immediate usage. At a future date the Stream registry will live on the blockchain.
+
+Please remember, you may need to escape the pathname slash depending on your coding environment.
+
+<Image
```

Comment:
Note the borders don't really match the rounded corners of the image:

![Screenshot 2021-02-18 at 10 22 48](https://user-images.githubusercontent.com/1064982/108326810-5ac79700-71d3-11eb-823f-b38c93cf2fc9.png)